Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ruby:2.5.1-alpine
+MAINTAINER apply for legal aid team
+
+RUN apk --update add --virtual build-dependencies \
+                                    build-base \
+                                    libxml2-dev \
+                                    libxslt-dev \
+#                                    postgresql-dev \
+                                    sqlite-dev \
+
+                                    tzdata \
+                                    && rm -rf /var/cache/apk/*
+
+RUN bundle config build.nokogiri --use-system-libraries
+
+RUN mkdir /usr/src/app
+
+WORKDIR /usr/src/app
+
+COPY . /usr/src/app
+
+RUN gem update bundler
```

Comment:
This line doesn't currently have any practical effect, as the base ruby image already contains bundler 1.16.3. Whether the line is there or not, after running a `docker build -t laa-apply-for-legalaid-api` this always shows the same output:

```$ docker run   laa-apply-for-legalaid-api gem list bundler
bundler (1.16.3)
```

However, if we leave it in then I think that we'll get the latest bundler version whenever it's released. I think what we should do is rely on the base image for bundler version bumps for now ie. remove this line.